packages/loader/container-loader/src/container.ts

```diff
@@ -306,7 +306,7 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 // It is also default mode in general.
                 const defaultMode: IContainerLoadMode = { opsBeforeReturn: "cached" };
                 assert(pendingLocalState === undefined || loadOptions.loadMode === undefined,
-                    "pending state requires immidiate connection!");
+                    0x1e0 /* "pending state requires immidiate connection!" */);
```

Comment:
can you also ensure this assert get the same shortcode in main, and is not already in use.

Reply:
Yeah will verify and make sure to integrate this change back to main as well

Reply:
ok. i'll approve, but make sure the shortcodes are aligned before release. our cross-release tooling isn't great. i'm thinking i'll require shortcodes for changes in release branches, as ideally its computed in main, and then backported to prevent duplicates